sqlglot/generator.py

```diff
@@ -1520,6 +1520,8 @@ def insert_sql(self, expression: exp.Insert) -> str:
         else:
             this = self.INSERT_OVERWRITE if overwrite else " INTO"
 
+        stored = self.sql(expression, "stored")
+        stored = f" {stored}" if stored else ""
```

Comment:
I'm a bit confused here, how is `STORED` being generated? Is the AST we're getting back correct?

Reply:
This is being parsed as `exp.FileFormatProperty` from `parser.py::_parse_stored()`, and Hive overrides it's generation in it's generator's `TRANSFORMS`